spec/integration/operations_spec.rb

```diff
@@ -86,6 +86,56 @@ module Hawkular::Operations::RSpec
       end
     end
 
+    it 'should run into error callback because bad hash parameters' do
```

Comment:
in combination with my comment above, I recommend adding a test where callback is nil